site/en/docs/extensions/mv3/manifest/homepage_url/index.md

```diff
@@ -2,14 +2,18 @@
 layout: "layouts/doc-post.njk"
 title: "Manifest - Homepage Url"
 date: 2013-05-12
-updated: 2018-04-26
+updated: 2023-01-04
 description: Reference documentation for the homepage_url property of manifest.json.
 ---
 
-The URL of the homepage for this extension. The extensions management page (chrome://extensions)
-will contain a link to this URL. This field is particularly useful if you [host the extension on
-your own site][1]. If you distribute your extension using the [Chrome Web Store][2], the homepage
-URL defaults to the extension's own page.
+An optional Manifest key containing a String for a valid homepage URL. Developers may choose to set the extension's homepage to their personal or company's website. If the parameter is left undefined, the default homepage will be the extension's Chrome Web Store page listed on the extensions management page (chrome://extensions). This field is particularly useful if you [host the extension on your own site][1]. 
 
-[1]: /docs/extensions/mv3/hosting
-[2]: https://chrome.google.com/webstore
+ ```json
+{
+  // ...
+  "homepage_url": "https://validURL.com,",
```

Comment:
Nit: Would it be better to use "https://path/to/homepage" to be consistent with https://developer.chrome.com/docs/extensions/mv3/manifest/ on `main`?

Alternatively we could use something like https://example.com (since that's a more formal looking example page) and make sure to update the manifest page too.

Reply:
We should be example.com since the W3C has declared it off limits. Someone could register validURL.com and put up content we don't like.